users/ritvikkolhe/Task 3/Quiz.html

```diff
@@ -0,0 +1,52 @@
+        <br><br><h2>Welcome to IEEE Quiz</h2><br><br>
+        <h4>Q.1 Which of the following is the best club/chapter in VIT ?</h4>
+        <input type="radio" name="" onclick="add()" value="Correct"> IEEE-VIT<br>
+        <input type="radio" name="" value="Incorrect"> CSI<br>
+        <input type="radio" name="" value="Incorrect"> GDG<br>
+        <input type="radio" name="" value="Incorrect"> ACM<br><br>
+        <h4>Q.2 Total number of people in IEEE-VIT family ?</h4>
+        <input type="radio" name="" value="Incorrect"> 70<br>
+        <input type="radio" name="" value="Incorrect"> 71<br>
+        <input type="radio" name="" value="Incorrect"> 75<br>
+        <input type="radio" name="" onclick="add()" value="Correct"> 74<br><br>
```

Comment:
Well the logic here is little flawed, if I click on the option again and again then `count` will increase, allowing me to get more than `5 Marks`. Once you start checking all the answers only when `Submit` is clicked, this issue will be resolved